go/private/actions/cover.bzl

```diff
@@ -55,6 +55,8 @@ def emit_cover(go, source):
         args.add("-var", cover_var)
         args.add("-src", src)
         args.add("-srcname", srcname)
+
+        # TODO: should we set this to atomic by default?
```

Comment:
Yes, I think so. `"atomic"` is a more sensible default than `"set"`.

This code path is mostly dead, though it may be used through the toolchain API by rules outside of rules_go.

Reply:
Should I use "atomic" in compilepkg even if `go.mode.race` is not set?

Reply:
Ah, sorry, scratch my last comment. Not thinking clearly today.

Let's do the same thing as in `compilepkg.bzl`: change the mode based on `go.cover.mode`.